### ISSUE TITLE
FetchError: request to https://..../WsWorkunits/WUQuery.json failed, reason: unable to get local issuer certificate

### ISSUE THREAD

**Issue by lpezet** [hpcc-systems/vscode-ecl#286]
Hello!

Sigh, I'm back at it again.
This time I have a CA-issued SSL certificate (from AWS) and yet I can't see my workunit result in the "HPCC" panel.
When I opened the Webview from the Developer Tools, I saw this error:

```
FetchError: request to https://..../WsWorkunits/WUQuery.json failed, reason: unable to get local issuer
```
None of the tricks we mentioned in https://github.com/hpcc-systems/vscode-ecl/issues/271 work.
All I can find googling this error is http proxy (which I don't use) and git related stuff.

Any ideas?

**Comment by GordonSmith** [comment]
@lpezet - would it be possible to get access to this setup to test against?

**Comment by lpezet** [comment]
@GordonSmith That is correct. I can submit just fine, I just can't see the results in the "HPCC" panel (I just get the spinning "Loading" animation forever).
This is a work setup so, as is, it's not possible to grant access to it. I'll try to re-create it so I can grant access to it.
Thanks for helping out @GordonSmith!

**Comment by GordonSmith** [comment]
@lpezet what server version are you using - I am scratching my head as to why this is not working...  In theory the new version should have a better chance of working with self signed certs than the older version.
